_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Alerting dashboards and visualizations 
+parent: Alerting
+nav_order: 50
+---
+
+# Alerting dashboards and visualizations
+Introduced 2.9
+{: .label .label-purple }
+
+Create, manage, and take action on your alerts in a single, consolidated view and identify and resolve issues quickly. Use the **Dashboard** interface to:
+
+- Set up, add, and adjust rules and conditions that trigger alerts and notifications.
+- Create graphs that show trends and patterns and build intuitive dashboards to stay on top of important metrics and data points in real time.
+- Monitor your alerts in one place with at-a-glance views.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/alerting-viz.png" alt="Example alerting visualization" width="800" height="800">
+
+## Getting started 
+
+Before getting started, you must have:
+
+- Installed OpenSearch and OpenSearch Dashboards version 2.8 or later. See [Installing OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/).
+- Installed Alerting and Notifications Dashboards plugins. See [Managing OpenSearch Dashboards plugins]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/plugins/) to get started.
+
+## Configuring admin settings
+
+Users can only access, create, or manage alerts for resources for which they have permissions. Access to alerting dashboards and visualizations is controlled by OpenSearch and OpenSearch Dashboards permissions. It is enabled by default and appears as a feature under **Dashboards Management** > **Advanced Settings** > **Visualization**. If the setting is disabled, it does not appear. You can disable the setting at the cluster level through the `opensearch-dashboards.yml` file.
+
+## General requirements for alerting visualizations
+
+Alerting visualizations are displayed as time-series charts that give you a snapshot of the alert, alert status, last updated time, and reason for the alert. You can show up to 10 metrics on your chart, and each series can be shown as a line in the chart.
+
+Keep in mind the following requirements when setting up or creating alerting visualizations:
+
+- Must be a [Vizlib line chart](https://community.vizlib.com/support/solutions/articles/35000107262-vizlib-line-chart-introduction)
```

Comment:
It would be preferable for these to be structured as complete sentences.

Reply:
Revised